gcp/modules/couchdb/main.tf

```diff
@@ -64,3 +68,39 @@ resource "null_resource" "couchdb_finish_cluster" {
     EOF
   }
 }
+
+resource "null_resource" "couchdb_enable_pv_backups" {
+  depends_on = ["module.couchdb"]
+
+  triggers = {
+    couchdb_replicas = "${var.couchdb_replicas}"
+    backup_deltas    = "${var.backup_deltas}"
+  }
+
+  provisioner "local-exec" {
+    command = <<EOF
+      # We have to rely on jq for filtering, since it is not possible to use regexp in jsonpath:
```

Comment:
Unfortunately, yes. There is no way to filter output using only `jsonpath`.

Reply:
Ok. I don't understand why the comment is talking about regexp though? Maybe you can clarify it?

Reply:
Changed for `We have to rely on jq for filtering, since it is not possible to use regexp or string functions in kubectl jsonpath`

Reply:
I still don't understand why this comment talks about regexp. Can you explain that part (or remove the reference, since I don't see any regular expressions anywhere in this PR)?

Reply:
We want to get all PVs claimed by CouchDB. And all CouchDB PVCs got unique names with `database-storage-couchdb` as common part. To apply this condition in [JSONPath filter](https://kubernetes.io/docs/reference/kubectl/jsonpath/) we need either regexp match (`=~`) or string function (substring, i.e. startsWith) and none of these available in kubectl jsonpath implementation.

Reply:
Done. What about stg/prd Terragrunt modules for CouchDB? I was about to add them as well.